integration-cli/docker_cli_swarm_test.go

```diff
@@ -879,7 +879,7 @@ func (s *DockerSwarmSuite) TestUnlockEngineAndUnlockedSwarm(c *check.C) {
 
 	// unlocking an unlocked swarm should return an error
 	cmd = d.Command("swarm", "unlock")
-	cmd.Stdin = bytes.NewBufferString("wrong-secret-key")
+	// it does not even ask for the key - it just errors
```

Comment:
This should check that the output is not prompting for a key (we can do that in the assertion below by changing `checker.Contains` to something that maches the exact output).